app/models/service_provider.rb

```diff
@@ -180,4 +181,16 @@ def contains_invalid_attribute?
     possible_attributes = ALLOWED_IAL1_ATTRIBUTES + ALLOWED_IAL2_ATTRIBUTES
     attribute_bundle.any? { |att| !possible_attributes.include?(att) }
   end
+
+  def redirect_uris_contains_no_wildcards
```

Comment:
I dont like repeating this code just bc the validator gem is generic. I'll be going back and adding specificity to the errors returned from the validator. Hold off on review.

Reply:
Refactor incoming: https://github.com/18F/identity-validations/pull/19